src/Authorization/FitOnFhir.Authorization/Config/AuthenticationConfiguration.cs

```diff
@@ -0,0 +1,56 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace FitOnFhir.Common.Config
+{
+    public class AuthenticationConfiguration
+    {
+        private string[] _providerMetadataEndpoints;
+
+        /// <summary>
+        /// Indicates whether anonymous logins are allowed.
+        /// </summary>
+        public bool IsAnonymousLoginEnabled { get; set; }
+
+        /// <summary>
+        /// The URL that any auth tokens are granted for.
+        /// </summary>
+        public string Audience { get; set; }
+
+        /// <summary>
+        /// The list of user allowed identity providers.
+        /// </summary>
+        public virtual IEnumerable<string> IdentityProviderMetadataEndpoints => _providerMetadataEndpoints;
```

Comment:
nit: I don't think these are "metadata endpoints" yet (because they haven't been appended with `/.well-known/openid-configuration`. I think maybe TokenAuthorities? To match the OpenIdConfigurationProvider authority param?